pkg/internal/goexec/gofile.go

```diff
@@ -20,6 +40,12 @@ func findLibraryVersions(elfFile *elf.File) (map[string]string, error) {
 
 	goVersion = strings.ReplaceAll(goVersion, "go", "")
 	log().Debug("Go version detected", "version", goVersion)
+
+	if !supportedGoVersion(goVersion) {
+		log().Debug("Unsupported Go version detected. Skipping instrumentation", "version", goVersion)
```

Comment:
This line is not required, as the error already describes the message and will be logged later.

Reply:
No problem. I'll drop it.